SparkPost/2web2ui
# SA-1196: Health Score display for subaccount reporting users

### What Changed
 -Selects the correct health score data for subaccount reporting user.

### How To Test
 - Create suabccount Reporting user in an account w/ signals data
 - Go to `/signals/health-score`. Verify that the health score dashboard contains data calculated for that subaccount only.
